src/redux/sagas.js

```diff
@@ -6,7 +6,15 @@ import { requestSpendingsSuccess, requestSpendingsError } from './actions';
 
 function* getSpendingsWorker(data) {
   try {
-    const response = yield call(spendings.getSpendings, data.payload.page, data.payload.perPage);
+    const response = yield call(
+      spendings.getSpendings,
```

Comment:
It's easier to retrieve payload values and only then use them, e.g.:
```js
const { page, perPage, ... } = action.payload`
```
Also, please rename `data` parameter to `action`. All sagas receive `actions` not `data`.